src/module/actors/character-sheet.ts

```diff
@@ -247,7 +248,7 @@ export class CharacterActorSheet extends ActorSheet<
 
     this.popUpDialog = new ValidatedDialog(
       {
-        failCallback: (button: ButtonData): void => {
+        failCallback: (): void => {
```

Comment:
Again, no random changes outside the idea of the PR.

Reply:
Done as part of #32.